engines/scumm/insane/insane.cpp

```diff
@@ -1383,7 +1383,9 @@ void Insane::smlayer_setActorFacing(int actornum, int actnum, int frame, int dir
 
 const char *Insane::handleTrsTag(int32 trsId) {
 	debugC(DEBUG_INSANE, "Insane::handleTrsTag(%d)", trsId);
-	return _player->getString(trsId);
+	if (_player != nullptr) {
+		return _player->getString(trsId);
+	} else return nullptr;
```

Comment:
Try adding a warning here, like "Couldn't load Trs block corresponding to id {trsId}, are you maybe missing a TRS subtitle file?"

Reply:
Done